### ISSUE TITLE
SharePoint list (Team Request Settings) is empty

### ISSUE THREAD

**Issue by Rowdy** [OfficeDev/microsoft-teams-apps-requestateam#40]
I just ran the script and unfortunately the SharePoint list is empty :-(
I get an error during the script but it finished as supposed to.
<img width="828" alt="Screenshot 2020-05-26 at 11 14 29" src="https://user-images.githubusercontent.com/309042/82886125-5e7d0480-9f46-11ea-9fcb-b1ce6d09a138.png">

**Comment by alexc-MSFT** [comment]
> I just ran the script and unfortunately the SharePoint list is empty :-(
> I get an error during the script but it finished as supposed to.
> <img alt="Screenshot 2020-05-26 at 11 14 29" width="828" src="https://user-images.githubusercontent.com/309042/82886125-5e7d0480-9f46-11ea-9fcb-b1ce6d09a138.png">

Please can you tell me what version of the PnP PowerShell that you have please?

Thanks

**Comment by Rowdy** [comment]
PS C:\Users\John> Get-Module -Name SharePointPnPPowerShellOnline -ListAvailable | Select Name,Version

Name                          Version
----                          -------
SharePointPnPPowerShellOnline 3.21.2005.1
SharePointPnPPowerShellOnline 3.21.2005.1

**Comment by Rowdy** [comment]
`**TEAMS REQUESTS SITE CREATION COMPLETE**
### REQUESTS SPO SITE CONFIGURATION ###
Configuring SharePoint site...
Applying provisioning template...
Apply-PnPProvisioningTemplate : The object id
"site:ad6c95f7-9442-4996-8ab2-6b4e9ebf9350:web:a43db0f3-3899-435c-a7f4-a19ea7306140" is invalid.
In C:\Users\Tom\Downloads\microsoft-teams-apps-requestateam-master\microsoft-teams-apps-requestateam-master\Deployment\
Scripts\deploy.ps1:285 Zeichen:9
+         Apply-PnPProvisioningTemplate -Path (Join-Path $packageRootPa ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : WriteError: (:) [Apply-PnPProvisioningTemplate], ServerException
    + FullyQualifiedErrorId : EXCEPTION,SharePointPnP.PowerShell.Commands.Provisioning.Site.ApplyProvisioningTemplate
Applied template
Error occured while configuring the SharePoint site: Ausnahme beim Aufrufen von "ExecuteQuery" mit 0 Argument(en):  "The object id "site:ad6c95f7-9442-4996-8ab2-6b4e9ebf9350:web:a43db0f3-3899-435c-a7f4-a19ea7306140" is invalid."
### AZURE RESOURCES DEPLOYMENT ###
Starting Azure resources deployment...`

Different error message :-(

**Comment by Ochechule** [comment]
The same here... Settings list is empty. Install script returned error.

![image](https://user-images.githubusercontent.com/65981697/82951470-54690d80-9ffb-11ea-86e4-d55ca31f4b63.png)

PS C:\WINDOWS\system32> Get-Module -Name SharePointPnPPowerShellOnline -ListAvailable | Select Name,Version

Name                          Version
----                          -------
SharePointPnPPowerShellOnline 3.21.2005.1
SharePointPnPPowerShellOnline 3.20.2004.0
SharePointPnPPowerShellOnline 3.19.2003.0
SharePointPnPPowerShellOnline 3.18.2002.0
SharePointPnPPowerShellOnline 3.17.2001.2
SharePointPnPPowerShellOnline 3.15.1911.0
SharePointPnPPowerShellOnline 3.11.1907.0
SharePointPnPPowerShellOnline 3.10.1906.0
SharePointPnPPowerShellOnline 3.9.1905.3

**Comment by alexc-MSFT** [comment]
@Rowdy  @Ochechule Can you remove all of the installed PnP versions with Uninstall-Module SharePointPnPPowerShellOnline -AllVersions -Force 

And reinstall the latest version please

Then have another try and see what happens.

Thanks

**Comment by chetferg** [comment]
We are experiencing the same issues as well. Error generated from PS Script:

***Error occured while configuring the SharePoint site: Cannot convert argument "parameters", with value: "Microsoft.SharePoint.Client.ListItemCreationInformation", for "AddItem" to type "Microsoft.SharePoint.Client.ListItemCr
eationInformation": "Cannot convert the "Microsoft.SharePoint.Client.ListItemCreationInformation" value of type "Microsoft.SharePoint.Client.ListItemCreationInformation" to type "Microsoft.SharePoint.Client.ListItemCreation
Information"."***

Below is the only module I have installed for SharepointPnPPowerShellOnline
SharePointPnPPowerShellOnline 3.21.2005.2

**Comment by chetferg** [comment]
All list columns seem to be there judging by the screenshots. Are the list items something I can manually enter if the information can be provided?

**Comment by Ochechule** [comment]
- Uninstalling all PnP versions with Uninstall-Module command, 
- uninstalling SharePoint Online Management Shell through Control Panel, 
- manually deleting all left subfolders in c:\Program Files\WindowsPowerShell\Modules\Microsoft.Online.SharePoint.PowerShell and 
- running the deploy script again

FIXED the issue. _Team Request Settings_ and _Teams Templates_ lists now contain items. Thank you @alexc-MSFT.

**Comment by alexc-MSFT** [comment]
@Ochechule excellent, glad to here it's worked. I think it might be something to do with an older version of the SPO Management Shell combined with a newer version of PnP. Both of these should provide the Microsoft.SharePoint.Client dll which is used in the CSOM code.

@chetferg please can you also try the above?

**Comment by Rowdy** [comment]
> * Uninstalling all PnP versions with Uninstall-Module command,
> * uninstalling SharePoint Online Management Shell through Control Panel,
> * manually deleting all left subfolders in c:\Program Files\WindowsPowerShell\Modules\Microsoft.Online.SharePoint.PowerShell and
> * running the deploy script again
> 
> FIXED the issue. _Team Request Settings_ and _Teams Templates_ lists now contain items. Thank you @alexc-MSFT.

Did you install any Microsoft.Online.SharePoint.PowerShell or SharePointPnPPowerShellOnline module?

**Comment by Ochechule** [comment]
No I didn't. I just uninstalled everything and let the script to install its version of PnP. 


________________________________

**Comment by chetferg** [comment]
@alexc-MSFT @Rowdy @Ochechule  The issue has been resolved by doing the following:
-Uninstalling SPO Managment shell from Control Panel & Uninstall-Module
-Uninstalling SP PnP Versions 
-Letting the script reinstall these modules
I believe it was due to an outdated version of the SPO Management Shell (I was running 16.0.8316.0).

I did get these errors due to using the Automation Account that was already in place for this. Since I chose to Overwrite the site in the script I imagine this shouldn't be an issue, just wanted to make you all aware! Thanks for the help!

![Automation](https://user-images.githubusercontent.com/57763795/83142962-3e248580-a0bf-11ea-959b-42df5e04760b.JPG)
